### PR TITLE
Re-structuring project with separate package for each command's functions  

### PR DESCRIPTION
**Pull request by aayushKumarJarvis**
@ashwanthkumar : please review

### CODE REVIEW

File: cmd/trigger/trigger_http.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package ops
+package httpTrigger
```
Comment (ashwanthkumar) [inline cmd/trigger/trigger_http.go]: @aayushKumarJarvis This should be called `trigger`

File: cmd/trigger/trigger.go [hunk 2]
```diff
@@ -1,6 +1,9 @@
-package ops
+package httpTrigger
```
Comment (ashwanthkumar) [inline cmd/trigger/trigger.go]: Similarly here, this should be called `trigger`

File: cmd/test-utils/utils.go [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package ops
+package testutils
```
Comment (ashwanthkumar) [inline cmd/test-utils/utils.go]: @aayushKumarJarvis Can we make the directory name also as `testutils` instead of `test-utils`?

File: glide.lock [hunk 4]
```diff
@@ -1,40 +0,0 @@
-hash: 3349fcf3073f65831f119748aea95a0d7313ab9ed959b4b1b31cb13bf3da5d54
```
Comment (ashwanthkumar) [inline glide.lock:1]: @aayushKumarJarvis We need the `glide.lock` file.

File: Makefile [hunk 5]
```diff
@@ -30,7 +30,11 @@ all:
 
 test:
 	go test -v github.com/ind9/rocks
-	go test -v github.com/ind9/rocks/ops
+	go test -v github.com/ind9/rocks/cmd/backup
+	go test -v github.com/ind9/rocks/cmd/restore
+	go test -v github.com/ind9/rocks/cmd/statistics
+	go test -v github.com/ind9/rocks/cmd/consistency
+	go test -v github.com/ind9/rocks/cmd/compact
```
Comment (ashwanthkumar) [inline Makefile:38]: @aayushKumarJarvis the `trigger` and `cmd` packages are missing

**Comment by ashwanthkumar** [comment]
@aayushKumarJarvis Please check the build failure and merge conflicts.

**Comment by aayushKumarJarvis** [comment]
Finally 😄  😅

**Comment by aayushKumarJarvis** [comment]
@ashwanthkumar , with the recent cleanups we are doing, shall I delete the logrus-integration branch and make a fresh log integration support in the clean-up branch ?

The only problem I see is that the PR will be too heavy with almost 50 commits !

**Comment by aayushKumarJarvis** [comment]
@ashwanthkumar : Reverting from changes introduced by logrus integration branch to this branch. Branch is now stable with all other features except for logging. Please review

**Pull request merged**